### ISSUE TITLE
Include "Total" stat for Armor

### ISSUE THREAD

**Issue by bhollis** [Bungie-net/api#1071]
This is silly because we could just add up the stats, but it'd actually be really convenient to have a "Total" stat come down like how it's displayed in game. Having it come from the API/definitions would mean we wouldn't have to special-case this stat for localization, etc.

**Comment by vthornheart-bng** [comment]
Yeah, I dig that - I'll hit up some folks and see what's up with that stat.  It wouldn't be the first stat the game has that's an aggregate, so there's some precedent.  If they aren't interested, we could fake it on our side.

**Comment by floatingatoll** [comment]
TIL there’s a total stat in game

**Comment by vthornheart-bng** [comment]
Admittedly I hadn't noticed it either!  I had to message Ben separately to get clarification!

**Comment by vthornheart-bng** [comment]
TFS 838066

**Comment by bhollis** [comment]
Having a localized definition for the total stat would still be appreciated.

**Comment by bhollis** [comment]
Having a localized definition for the total stat would still be appreciated.
